packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -122,40 +165,103 @@
     &__jurisdiction-checkbox {
         display: flex;
         justify-content: center;
+        margin-bottom: 2rem;
         @include mobile {
             padding: 0.8rem;
         }
     }
 
     &__over-header {
         background-color: #dfeaff;
-        top: -1rem;
+        top: -2rem;
         left: 1.4rem;
         z-index: 10;
         width: 90%;
         height: 4rem;
         position: absolute;
-        border-radius: 4px;
+        border-radius: 8px;
         margin-bottom: 1rem;
-
+        border: 1px solid var(--text-info-blue);
         & > p {
             font-size: 16px;
             text-align: center;
         }
     }
+}
 
-    &__bullet-wrapper {
+.cfd-card-back-section {
+    &-items-container {
         display: flex;
-        margin-bottom: 1rem;
+        flex-direction: column;
+        gap: 1rem;
+    }
+
+    &-main {
+        height: 100%;
+    }
+
+    &-items-sub-container {
+        display: flex;
+        align-items: flex-start;
+        gap: 1rem;
+    }
+
+    &-back-button {
+        transition: transform 0.25s linear;
+        &:hover {
+            transform: translateX(-4px);
+        }
+    }
+}
 
-        &--checkmark {
-            margin-right: 1rem;
+.cfd-card-section {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    gap: 0.8rem;
+    &-divider {
+        width: 100%;
+        border-bottom: 2px solid var(--border-divider);
+    }
+    &-title-indicator {
+        padding: 0.6rem 1.2rem;
+        border-radius: $BORDER_RADIUS;
+        &__red-darker {
+            background: var(--modal-indicator-red-darker);
+        }
+        &__red-dark {
+            background: var(--modal-indicator-red-dark);
+        }
+        &__red-light {
+            background: var(--modal-indicator-red-light);
+        }
+        &__yellow-dark {
+            background: var(--modal-indicator-yellow-dark);
+        }
+        &__yellow-light {
+            background: var(--modal-indicator-yellow-light);
+        }
+        &__violet-dark {
+            background: var(--modal-indicator-violet-dark);
+        }
+        &__brown-dark {
+            background: var(--modal-indicator-brown-dark);
         }
```

Comment:
What is this code for?
Can we rely on our theme to change colors?

Reply:
these are used for setting the box color based on the card being displayed, and these colors are defined in common(outside light and dark classes) area of themes.scss since the same colors are used in light and dark mode.
shall we make any changes here?
<img width="92" alt="Screenshot 2023-04-10 at 7 40 57 AM" src="https://user-images.githubusercontent.com/122449658/230832076-5c207ff4-ee27-48f2-a74e-ab8d210c3554.png">